api/api.go

```diff
@@ -46,6 +46,8 @@ type Authority interface {
 	GetRoots() (federation []*x509.Certificate, err error)
 	GetFederation() ([]*x509.Certificate, error)
 	Version() authority.Version
+	GenerateCertificateRevocationList() error
```

Comment:
GenerateCertificateRevocationList is not used in the api package, it should not be here.